sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -541,12 +541,55 @@ def test_binomial_rewrite():
     k = Symbol('k', integer=True)
     x = Symbol('x')
 
-    assert binomial(n, k).rewrite(
-        factorial) == factorial(n)/(factorial(k)*factorial(n - k))
-    assert binomial(
-        n, k).rewrite(gamma) == gamma(n + 1)/(gamma(k + 1)*gamma(n - k + 1))
+    assert binomial(n, k).rewrite(factorial) == factorial(n)/(factorial(k)*factorial(n - k))
+    assert binomial(n, k).rewrite(gamma) == gamma(n + 1)/(gamma(k + 1)*gamma(n - k + 1))
     assert binomial(n, k).rewrite(ff) == ff(n, k) / factorial(k)
     assert binomial(n, x).rewrite(ff) == binomial(n, x)
+    assert binomial(n, k).rewrite(gamma).rewrite('tractable') == exp(-loggamma(k + 1))*exp(loggamma(n + 1))*exp(-loggamma(-k + n + 1))
```

Comment:
please wrap. It is always safe to put a line break within a grouping character (
like parentheses)